plugins/verifier/sample/sample.go

```diff
@@ -37,7 +39,18 @@ type PluginInputConfig struct {
 }
 
 func main() {
+	// create a plugin logger
+	pluginlogger := logger.NewLogger()
+
+	// output to stderr
+	pluginlogger.Info("initialized plugin")
+	pluginlogger.Debugf("plugin %s %s", "sample", "1.0.0")
+
 	skel.PluginMain("sample", "1.0.0", VerifyReference, []string{"1.0.0"})
+
+	// set output to stdout
+	pluginlogger.SetOutput(os.Stdout)
```

Comment:
wondering when should we use std out vs err

Reply:
I see, thanks for the clarification. :) if we could just add these comment to the sample would be good. Exactly what you have mentioned. " stderr is the default for messages, but we do provide the option  to redirecting to stdout"